apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -233,35 +233,43 @@ export default function LearningGoal({
               {UNDERSTANDING_LEVEL_STRINGS[displayUnderstanding]}
             </BodyThreeText>
           )}
-          <div className={style.submittedFeedback}>
-            {submittedEvaluation?.understanding && (
-              <BodyThreeText>
-                {UNDERSTANDING_LEVEL_STRINGS[submittedEvaluation.understanding]}
-              </BodyThreeText>
-            )}
-            {submittedEvaluation?.feedback && (
-              <FontAwesome
-                icon="message"
-                className="fa-regular"
-                title={i18n.feedback()}
-              />
-            )}
-          </div>
+          {submittedEvaluation && (
+            <div className={style.submittedEvaluation}>
+              {submittedEvaluation?.understanding && (
```

Comment:
Do you need the `?` here since we have the `submittedEvaluation &&` above?  Really just a small question and not stopping us from merging this, I think

Reply:
true! I don't -- just left it from before